lib/src/parse/stylesheet.dart

```diff
@@ -2909,12 +2917,25 @@ abstract class StylesheetParser extends Parser {
           scanner.state = beforeArguments;
           return null;
         }
+        return CalculationExpression(name, arguments, scanner.spanFrom(start));
 
+      case "pow":
+      case "log":
+      case "atan2":
+        var arguments = _calculationArguments(2);
         return CalculationExpression(name, arguments, scanner.spanFrom(start));
 
       case "clamp":
         var arguments = _calculationArguments(3);
         return CalculationExpression(name, arguments, scanner.spanFrom(start));
+      case "round":
+        var beforeArguments = scanner.state;
+        var arguments = _calculationArguments(3);
+        if (arguments.length == 1) {
```

Comment:
I believe this would mean that code like:
```css
a {
  --args: up, 36px, 5px;
  b: round(var(--args));
}
```
would error since it would try to treat all one-argument `round`s as calls to the Sass global function. `round` should probably parse the same way as `clamp`, with the legacy behavior handled in the calc simplification during evaluation. I'll get a spec update out soon for handling the backwards compatibility.